.env.sample

```diff
@@ -1,2 +1,6 @@
-NEXT_PUBLIC_APP_URL = 'http://localhost:3000'
-GOOGLE_ANALYTICS_ID = 'UA-XXXXXXXXX-X'
+NEXT_PUBLIC_APP_URL='http://localhost:3000'
+GOOGLE_ANALYTICS_ID='UA-XXXXXXXXX-X'
+NEXT_PUBLIC_APP_API_BASE_URL='http://localhost:8000/v2'
+NEXT_PUBLIC_APP_API_STATIC_URL='http://localhost:8000/static/v2'
+NEXT_PUBLIC_APP_MAP_URL='https://v2.map.aq.sensors.africa'
+NEXT_PUBLIC_APP_API_TOKEN=''
```

Comment:
1. Sort these alphabetically
2. Secrets such as tokens shouldn't have `NEXT_PUBLIC_`; This prefix tells Next.js to make sure the values are available on the client-side as well (which we don't want).